diff --git a/dist/API.d.ts b/dist/API.d.ts
index 3fad2ed9f4412d5421b212abdc756a64eff7ce13..c5528171d38dd84a949e14f0d903eece2bba2c2e 100644
--- a/dist/API.d.ts
+++ b/dist/API.d.ts
@@ -15,6 +15,9 @@ export declare class API {
     getV2: (endpoint: string, params?: {
         [key: string]: any;
     }) => Promise<any>;
+    deleteV2: (endpoint: string, params?: {
+        [key: string]: any;
+    }) => Promise<any>;
     getWebsite: (endpoint: string, params?: {
         [key: string]: any;
     }) => Promise<any>;
diff --git a/dist/API.js b/dist/API.js
index d92a54c425a37ce5269a129570b6d515dede7240..2625c9486eeb9f028bf0aea415898e7d4c4295bd 100644
--- a/dist/API.js
+++ b/dist/API.js
@@ -59,6 +59,7 @@ var API = /** @class */ (function () {
         this.getWebsite = function (endpoint, params) { return _this.getRequest(webURL, endpoint, params); };
         this.getURL = function (URI, params) { return _this.fetchRequest(URI, "GET", params); };
         this.post = function (endpoint, params) { return _this.fetchRequest("".concat(apiURL, "/").concat(endpoint), "POST", params); };
+        this.deleteV2 = function (endpoint, params) { return _this.fetchRequest("".concat(apiV2URL, "/").concat(endpoint), "DELETE", params); };
         this.options = function (method, params) {
             var options = {
                 method: method,
